File: app/src/main/java/com/mkrworld/mobilpay/provider/fragment/FragmentTag.java
package com.mkrworld.mobilpay.provider.fragment;

import com.mkrworld.mobilpay.ui.fragment.FragmentForgotPassword;

/**
 * Created by mkr on 13/3/18.
 * Interface to hold the tag of the Fragment
 */
public interface FragmentTag {
    String MERCHANT_LOGIN = "FragmentMerchantLogin";
    String MERCHANT_HOME = "FragmentMerchantHome";
    String MERCHANT_QR_CODE = "FragmentMerchantQrCode";
    String MERCHANT_QR_CODE_GENERATOR = "FragmentMerchantQrCodeGenerator";
    String MERCHANT_AEPS_COLLECT = "FragmentMerchantAEPSCollect";
    String MERCHANT_SEND_BILL = "FragmentMerchantSendBill";
    String MERCHANT_COLLECTION_SUMMARY = "FragmentMerchantCollectionSummary";
    String FORGOT_PASSWORD = "FragmentForgotPassword";
    String CHANGE_PASSWORD = "FragmentChangePassword";
    String CHANGE_PASSWORD_BY_OTP = "FragmentChangePasswordByOtp";
}

File: app/src/main/java/com/mkrworld/mobilpay/provider/fragment/FragmentTag.kt
package com.mkrworld.mobilpay.provider.fragment

/**
 * Created by mkr on 13/3/18.
 * Interface to hold the tag of the Fragment
 */
interface FragmentTag {
    companion object {
        val LOGIN = "Login"
        val HOME = "FragmentHome"
        val CASH_COLLECT = "FragmentCashCollect"
        val QR_CODE_GENERATOR = "FragmentQrCodeGenerator"
        val AEPS_COLLECT = "FragmentAEPSCollect"
        val SEND_BILL = "FragmentSendBill"
        val QR_CODE = "FragmentQrCode"
        val COLLECTION_STATUS = "FragmentCollectionStatus"
        val FORGOT_PASSWORD = "FragmentForgotPassword"
        val CHANGE_PASSWORD = "FragmentChangePassword"
        val CHANGE_PASSWORD_BY_OTP = "FragmentChangePasswordByOtp"
        val COLLECTION_SUMMARY = "FragmentCollectionSummary"
        val UNPAID_DETAILS = "UnpaidDetails"
        val NOTIFICATION = "FragmentNotification"
        val NOTIFICATION_TAB_ALL = "FragmentNotificationTabAll"
        val NOTIFICATION_TAB_AGENT = "FragmentNotificationTabAgent"
        val NOTIFICATION_TAB_CUSTOMER = "FragmentNotificationTabCustomer"
    }
}
